lib/src/_util.dart

```diff
@@ -31,3 +31,8 @@ T cast<T>(value) {
     '$value is of type ${value.runtimeType} that is not subtype of $T',
   );
 }
+
+Object? safeToEncodable(Object? object) {
```

Comment:
```suggestion
Object? toEncodable(Object? object) {
```
This name reads a bit weird
Could this just be left as toEncodable? Otherwise a JSDoc may be nice here giving some context.